From b44261e5846f054b16901e6fe0d31d23cee8e74b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Tue, 7 Nov 2023 11:07:53 -0500
Subject: [PATCH] [PATCH] disable clang verbose build warnings

---
 src/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 84f98033a..8fda27e5b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -521,6 +521,8 @@ target_compile_options(
         $<$<CXX_COMPILER_ID:Clang,AppleClang>:-Wno-undefined-func-template>
         $<$<CXX_COMPILER_ID:Clang,AppleClang>:-Wno-unused-member-function>
         $<$<CXX_COMPILER_ID:Clang,AppleClang>:-Wno-unused-template>
+        $<$<CXX_COMPILER_ID:Clang,AppleClang>:-Wno-unknown-warning-option>
+        $<$<CXX_COMPILER_ID:Clang,AppleClang>:-Wno-unsafe-buffer-usage>
 
         # This warning was removed in Clang 13
         $<$<AND:$<CXX_COMPILER_ID:Clang,AppleClang>,$<VERSION_LESS:$<CXX_COMPILER_VERSION>,13.0>>:-Wno-return-std-move-in-c++11>
-- 
2.25.1

