From 51ed947d7189a52bc0dd0f68b20c7fb201d2821c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Sun, 22 Jan 2023 08:01:56 -0500
Subject: [PATCH 2/3] disable autoschedulers

---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3a89e132b..222b7335b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -636,8 +636,8 @@ endif()
 ##
 
 if (BUILD_SHARED_LIBS)
-    message(STATUS "Building autoschedulers enabled")
-    add_subdirectory(autoschedulers)
+    #message(STATUS "Building autoschedulers enabled")
+    #add_subdirectory(autoschedulers)
 else ()
     message(STATUS "Building autoschedulers disabled (static Halide)")
 endif ()
-- 
2.34.1

