From 37576a87da1ae331225231c56c8c36aa389ef383 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Sat, 21 Jan 2023 10:15:38 -0500
Subject: [PATCH 1/3] add HALIDE_HOST_TOOLS_DIR cmake option

---
 tools/CMakeLists.txt | 26 ++++++++++++++++++++------
 1 file changed, 20 insertions(+), 6 deletions(-)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 2d66e5271..25336723e 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -2,14 +2,28 @@
 # Build time tools
 ##
 
-add_executable(build_halide_h build_halide_h.cpp)
-target_compile_options(build_halide_h PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+if(DEFINED HALIDE_HOST_TOOLS_DIR)
+    add_executable(build_halide_h IMPORTED GLOBAL)
+    set_property(TARGET build_halide_h PROPERTY IMPORTED_LOCATION
+        "${HALIDE_HOST_TOOLS_DIR}/build_halide_h${CMAKE_EXECUTABLE_SUFFIX}")
 
-add_executable(binary2cpp binary2cpp.cpp)
-target_compile_options(binary2cpp PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+    add_executable(binary2cpp IMPORTED GLOBAL)
+    set_property(TARGET binary2cpp PROPERTY IMPORTED_LOCATION
+        "${HALIDE_HOST_TOOLS_DIR}/binary2cpp${CMAKE_EXECUTABLE_SUFFIX}")
 
-add_executable(regexp_replace regexp_replace.cpp)
-target_compile_options(regexp_replace PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+    add_executable(regexp_replace IMPORTED GLOBAL)
+    set_property(TARGET regexp_replace PROPERTY IMPORTED_LOCATION
+        "${HALIDE_HOST_TOOLS_DIR}/regexp_replace${CMAKE_EXECUTABLE_SUFFIX}")
+else()
+    add_executable(build_halide_h build_halide_h.cpp)
+    target_compile_options(build_halide_h PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+
+    add_executable(binary2cpp binary2cpp.cpp)
+    target_compile_options(binary2cpp PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+
+    add_executable(regexp_replace regexp_replace.cpp)
+    target_compile_options(regexp_replace PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/wd4996>)
+endif()
 
 ##
 # Interface target for enabling PNG/JPEG support in Halide
-- 
2.34.1

