From dfcb3a317b420e1e5b4f98506f0e9d12a4adb406 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Sun, 22 Jan 2023 14:40:59 -0500
Subject: [PATCH] disable imported clang+llvm executable promotion

---
 dependencies/llvm/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dependencies/llvm/CMakeLists.txt b/dependencies/llvm/CMakeLists.txt
index 248aaf9ea..a8e613a37 100644
--- a/dependencies/llvm/CMakeLists.txt
+++ b/dependencies/llvm/CMakeLists.txt
@@ -42,6 +42,8 @@ cmake_dependent_option(Halide_BUNDLE_LLVM "When built as a static library, inclu
 # Promote LLVM/Clang executable targets
 ##
 
+if(PROMOTE_IMPORTED_CLANG_LLVM)
+
 set_target_properties(llvm-as clang PROPERTIES IMPORTED_GLOBAL TRUE)
 
 # clang-tools-extra is optional, but provides the clang-format target
@@ -49,6 +51,8 @@ if (TARGET clang-format)
     set_target_properties(clang-format PROPERTIES IMPORTED_GLOBAL TRUE)
 endif ()
 
+endif()
+
 ##
 # Create options for including or excluding LLVM backends.
 ##
-- 
2.25.1

