From 00b2a9f9232285bcab3af7b2190fac2361952a89 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Tue, 18 Oct 2022 17:20:18 -0400
Subject: [PATCH] add llvm name prefix to llc,lli,opt tools

---
 llvm/tools/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/llvm/tools/CMakeLists.txt b/llvm/tools/CMakeLists.txt
index 770e46c4806f..a7c8bb058624 100644
--- a/llvm/tools/CMakeLists.txt
+++ b/llvm/tools/CMakeLists.txt
@@ -53,3 +53,7 @@ foreach(p ${LLVM_EXTERNAL_PROJECTS})
 endforeach(p)
 
 set(LLVM_COMMON_DEPENDS ${LLVM_COMMON_DEPENDS} PARENT_SCOPE)
+
+set_target_properties(llc PROPERTIES OUTPUT_NAME "llvm-llc")
+set_target_properties(lli PROPERTIES OUTPUT_NAME "llvm-lli")
+set_target_properties(opt PROPERTIES OUTPUT_NAME "llvm-opt")
\ No newline at end of file
-- 
2.25.1

