From 71ed296a76832517fc695fb42c626eb89b9bfd7c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Moreau?= <mamoreau@devolutions.net>
Date: Fri, 1 May 2026 09:24:29 -0400
Subject: [PATCH] add llvm name prefix to llc,lli,opt tools

---
 llvm/tools/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/llvm/tools/CMakeLists.txt b/llvm/tools/CMakeLists.txt
index 729797aa4..aa65698f5 100644
--- a/llvm/tools/CMakeLists.txt
+++ b/llvm/tools/CMakeLists.txt
@@ -67,6 +67,10 @@ endforeach(p)
 
 set(LLVM_COMMON_DEPENDS ${LLVM_COMMON_DEPENDS} PARENT_SCOPE)
 
+set_target_properties(llc PROPERTIES OUTPUT_NAME "llvm-llc")
+set_target_properties(lli PROPERTIES OUTPUT_NAME "llvm-lli")
+set_target_properties(opt PROPERTIES OUTPUT_NAME "llvm-opt")
+
 if (LLVM_TOOL_LLVM_DRIVER_BUILD)
   # This is explicitly added at the end _after_ all tool projects so that it can
   # scrape up tools from other projects into itself.
-- 
2.54.0.windows.1

